core/src/martian/core.cljc

```diff
@@ -107,8 +107,9 @@
 (defn explore
   ([martian] (mapv (juxt :route-name :summary) (:handlers (resolve-instance martian))))
   ([martian route-name]
-   (when-let [{:keys [parameter-aliases summary] :as handler} (find-handler (:handlers (resolve-instance martian)) route-name)]
+   (when-let [{:keys [parameter-aliases summary deprecated] :as handler} (find-handler (:handlers (resolve-instance martian)) route-name)]
      {:summary summary
+      :deprecated deprecated
```

Comment:
It would be cleaner to only merge the deprecated key when it is true, to avoid cluttering up the explore output with a load of `:deprecated? false` (or, in the case of swagger, `nil`)